uTensor/core/sdtensor.hpp

```diff
@@ -4,6 +4,12 @@
 #include "uTensor/core/vm.hpp"
 
 
+#ifdef COMPILE_GREENTEA
+static string tmpprefix = "/fs/tmp/";
+#else
+static string tmpprefix = "/tmp/";
+#endif
```

Comment:
Do you guys know what is identifier could be used to know the program in built on mbed ?  I need to make the tmp prefix variable portable, such as gtest, green-tea, mbed (non-test-environment).

Reply:
Try going to this page and search for mbed_config.h

https://docs.mbed.com/docs/mbed-os-handbook/en/5.2/advanced/config_system/

Reply:
I have an example in test_helper.h

Reply:
@mbartling  i use COMPILE_GREENTEA now, but i need to separate test and release environment. That is why i need some identifier defined in mbed.

Reply:
done